tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/storage-stack.json

```diff
@@ -0,0 +1,352 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "external-shared-storage template for test_update.py::test_dynamic_file_systems_update",
+  "Parameters": {
+    "PublicAz3": {
+      "Type": "String",
+      "Default": ""
+    },
+    "PublicAz2": {
+      "Type": "String",
+      "Default": ""
+    },
+    "Public": {
```

Comment:
What is Public? Is that a subnet id?
Let's clarify that in the parameter description

Reply:
Yes, I've changed the name of the parameter to the more descriptive name PublicSubnetId and will push that with my next commit